app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.java

```diff
@@ -200,6 +186,65 @@ public CommentsViewModel(final @NonNull Environment environment) {
       .take(1)
       .compose(bindToLifecycle())
       .subscribe(__ -> refresh.onNext(null));
+
+    final Observable<Update> update = projectOrUpdate.map(Either::right);
+
+    // NB: A temporary workaround for nextPage ping on empty state. Better fix is to filter out
+    // non-scrollable views in `RecyclerViewPaginator`.
+    final Observable<Void> nonEmptyNextPage = comments
```

Comment:
this is the temp solution for empty page `nextPage` pings until #60 merges, as that needs more testing but is not a blocker to merging this.